test/components/component_test.rb

```diff
@@ -111,7 +111,11 @@ def test_all_components_support_inline_styles
     default_args = { style: "width: 100%;" }
     COMPONENTS_WITH_ARGS.each do |component, args, proc|
       render_component(component, default_args.merge(args), proc)
-      assert_selector("[style='width: 100%;']", visible: :all)
+      if component == Primer::Alpha::Tooltip
+        assert_selector("[style='width: 100%;visibility: hidden']", visible: :all)
+      else
+        assert_selector("[style='width: 100%;']", visible: :all)
+      end
```

Comment:
`assert_selector` looks for an exact match. I couldn't find an alternate matcher. I found [assert_matches_style](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FMatchers:assert_matches_style) but its not available even when I upgraded capybara.
if reviewer has ideas here...